sys/net/transport_layer/tcp/tcp_hdr_print.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2017 smlng <s@mlng.net>, HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
```

Comment:
Yapp, but since the Copyright line usually contains the organization the person(s) work for, having the author in an `@author` tag gives them at least some credit.